litmus-portal/Makefile

```diff
@@ -0,0 +1,80 @@
+# Makefile for building Litmus Portal
+# Reference Guide - https://www.gnu.org/software/make/manual/make.html
+
+#
+# Internal variables or constants.
+# NOTE - These will be executed when any make target is invoked.
+#
+IS_DOCKER_INSTALLED = $(shell which docker >> /dev/null 2>&1; echo $$?)
+
+
+.PHONY: all
+all: deps lint-checks build push
+
+.PHONY: help
+help:
+	@echo ""
+	@echo "Usage:-"
+	@echo "\tmake all   -- [default] builds the litmus containers"
+	@echo ""
+
+.PHONY: deps
+deps: _build_check_docker
+
+_build_check_docker:
+	@echo "------------------"
+	@echo "--> Check the Docker deps" 
+	@echo "------------------"
+	@if [ $(IS_DOCKER_INSTALLED) -eq 1 ]; \
+		then echo "" \
+		&& echo "ERROR:\tdocker is not installed. Please install it before build." \
+		&& echo "" \
+		&& exit 1; \
+		fi;
+
+.PHONY: build
+build: litmus-portal-build
+
+litmus-portal-build:
+	@echo "------------------"
+	@echo "--> Build frontend and backend images" 
+	@echo "------------------"
+	docker build ./frontend -f frontend/Dockerfile -t gdsoumya/litmusportal-frontend:ci
```

Comment:
gdsoumya/litmusportal-frontend:ci --> litmuschaos/litmusportal-frontend:ci

Reply:
change it in all the places

Reply:
Made a push with the changes